plenoptic/synthesize/eigendistortion.py

```diff
@@ -45,6 +46,9 @@ def fisher_info_matrix_eigenvalue(y, x, v):
     """
     Fv = fisher_info_matrix_vector_product(y, x, v)
     lmbda = Fv.T @ v
+    if v.shape[1] > 1:
+        lmbda = torch.diag(lmbda)
+
```

Comment:
Can we get a brief explanation for why this is done? (one sentence comment woudl suffice)

Reply:
done.